Source/Blazorise/Components/Bar/Bar.razor.cs

```diff
@@ -131,7 +143,7 @@ protected override void Dispose( bool disposing )
                     NavigationManager.LocationChanged -= OnLocationChanged;
```

Comment:
I think its better to first unwire the events before starting the async operation. Goes for all dispose methods

Reply:
You're right. Gonna change that.